test/strategy/EnrichClientFromRemoteConfigStrategyTest.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+var stormpath = require('stormpath');
+
+var common = require('../common');
+var _ = common._;
```

Comment:
Common is just a helper that collects all the requires. Because sometimes, like in the case of assert, I don't want to have to be requiring `chai` each and every time.